packages/governance/test/swingsetTests/contractGovernor/test-governor.js

```diff
@@ -167,8 +167,8 @@ test.serial('brokenUpdateStart', async t => {
     // TODO: allow either message
     // 'vote rejected outcome: Error: (an object) was not a live payment for brand (an object). It could be a used-up payment, a payment for another brand, or it might not be a payment at all.',
     // 'update failed: Error: (an object) was not a live payment for brand (an object). It could be a used-up payment, a payment for another brand, or it might not be a payment at all.',
-    'vote rejected outcome: Error: A Zoe invitation is required, not "[Alleged: Zoe Invitation payment]"',
-    'update failed: Error: A Zoe invitation is required, not "[Alleged: Zoe Invitation payment]"',
+    'vote rejected outcome: Error: A Zoe invitation is required, not (an object)',
+    'update failed: Error: A Zoe invitation is required, not (an object)',
```

Comment:
If you're running with a `prepare-...` setup import suitable for testing, then it should already be setting `errorTaming: 'unsafe'` in which case you should not see these redactions. Is the vat these are coming from run in a non-testing-setup vat?

Reply:
It has 
```
import test from 'ava';
```
I've updated it.

Reply:
Hmm.   I changed that to
``` javascript
import { test } from '@agoric/swingset-vat/tools/prepare-test-env-ava.js';
```

expecting that to be what you meant, but that didn't fix it.

Reply:
Yes, `import { test } from '@agoric/swingset-vat/tools/prepare-test-env-ava.js';` is exactly what I had in mind. Separately, we need to understand why that did not fix it, and where the code that generated that error message is running.

But no more on this needs to be done in this PR. Resolving.